File: Houseclub/src/main/java/me/grishka/houseclub/api/BaseResponse.java
package me.grishka.houseclub.api;

public class BaseResponse {
    public boolean success;
    public String errorMessage;

    @Override
    public String toString() {
        return "BaseResponse{" +
                "success=" + success +
                ", errorMessage='" + errorMessage + '\'' +
                '}';
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/api/BaseResponse.kt
package me.grishka.houseclub.api

class BaseResponse {
    var success = false
    var errorMessage: String? = null
    override fun toString(): String {
        return "BaseResponse{" +
            "success=" + success +
            ", errorMessage='" + errorMessage + '\'' +
            '}'
    }
}